mimic/model/nova_objects.py

```diff
@@ -822,7 +822,7 @@ def request_delete(self, http_delete_request, server_id):
         server.update_status(u"DELETED")
         return b''
 
-    def request_action(self, http_action_request, server_id):
+    def request_action(self, http_action_request, server_id, absolutize_url):
```

Comment:
Can we change `absolutize_url` variable name to be non-verb such as `absolute_url` since absolutize is a verb and sounds like a method name.